Sources/TABResourceLoader/Protocols/Resource types/NetworkResourceType.swift

```diff
@@ -58,6 +58,9 @@ public protocol NetworkResourceType {
 
   /// The query items to be added to the url to fetch this resource
   var queryItems: [URLQueryItem]? { get }
+  
+  /// The time interval for the URLRequest for this resource
+  var requestTimeInterval: TimeInterval? { get }
```

Comment:
@CalvHobbes could you change this to `requestTimeoutInterval` please

Reply:
yes, of course. I thought that's the term I used 🤦🏼‍♀️